Formula/code-server.rb

```diff
@@ -27,7 +28,7 @@ class CodeServer < Formula
 
   def install
     node = Formula["node@16"]
-    system "yarn", "--production", "--frozen-lockfile"
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec), "--unsafe-perm"
     # @parcel/watcher bundles all binaries for other platforms & architectures
     # This deletes the non-matching architecture otherwise brew audit will complain.
     prebuilds = buildpath/"lib/vscode/node_modules/@parcel/watcher/prebuilds"
```

Comment:
Try removing the two `rmtree`s below, including this line.

Reply:
Thank you! I'll try that now.